src/kv/idbstore.js

```diff
@@ -1,14 +1,14 @@
 // @ts-check
 
-const RELAXED = { durability: "relaxed" };
-const OBJECT_STORE = "chunks";
+const RELAXED = {durability: 'relaxed'};
```

Comment:
Is this going to go away in a different PR?

Reply:
Either way. I'll do it in this PR but maybe keep the commits separate. Will see...